Publish to Bintray mirror fails on i18n-cache

When running `./bin/update-bintray-repo.sh` to publish RN and JSC to the Bintray mirror, the script fails because yarn doesn't find the `i18n-cache` folder (in the temp folder).

```
yarn run v1.13.0
$ node i18n-cache/index.js
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module '/private/var/folders/py/vdkc920j69b255sld66fbxt40000gp/T/tmp.pJLxwthZ/i18n-cache/index.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
```

--- a/bin/update-bintray-repo.sh
+++ b/bin/update-bintray-repo.sh
@@ -56,6 +56,7 @@ jfrog bt config
 echo "Running yarn install in '${TMP_WORKING_DIRECTORY}'..."
 cp "package.json" "${TMP_WORKING_DIRECTORY}/"
 cp "yarn.lock" "${TMP_WORKING_DIRECTORY}/"
+cp -Ra "i18n-cache" "${TMP_WORKING_DIRECTORY}/"
 cd "${TMP_WORKING_DIRECTORY}"
 yarn install --silent
 

